src/components/Article.jsx

```diff
@@ -0,0 +1,52 @@
+import { useState, useEffect } from "react";
+import { useParams } from "react-router-dom";
+import { getArticle } from "../utils/api";
+import Header from "./Header";
+import Comments from "./Comments";
+
+export default function Article() {
+  const [article, setArticle] = useState([]);
+  const { article_id } = useParams();
+
+  useEffect(() => {
+    getArticle(article_id).then((body) => {
+      setArticle(body);
+    });
+  }, []);
+
+  let formattedDate = "";
+
+  if (article.created_at) {
+    formattedDate = article.created_at.slice(0, 10);
+  } else
+    return (
+      <>
+        <Header />
+        <p>Loading article...</p>
+      </>
```

Comment:
I cant see your isLoading state here. Shouldn't you have one for for this? ^^^

Reply:
It doesn't need one as the condition is only true when the article has loaded. I've throttled my connection and it appears to be working as expected 😄